client/me/help/help-contact/index.jsx

```diff
@@ -277,24 +278,6 @@ class HelpContact extends React.Component {
 		return isEn && ! this.props.isDirectlyFailed;
 	};
 
-	getSupportVariation = () => {
-		const { ticketSupportEligible } = this.props;
-
-		if ( this.shouldUseHappychat() ) {
```

Comment:
`shouldUseHappyChat` and `shouldUseDirectly` will now be redundant after this PR's changes are merged. I'll follow up with another PR that cleans those and their dependencies up.

Reply:
Sounds good to me. 👍